docs/source/developers/guide/communication.rst

```diff
@@ -0,0 +1,133 @@
+.. Also add a link to other resources.
+
+
+.. _communication:
+
+*************
+Communication 
+*************
+
+**About the contributors**
+
+The group of contributors is full of experts, software engineers and core
+developers but also full of users, learners and enthusiasts that love doing
+what we do and we want to spread this enthusiasm to you also.
+
+We all have silly questions and we all need help lots of times.
+We encourage you to be open to communication and we will help as much as
+we can.
+
+Due to the scale of the project which includes many languages, everybody
+contributing will be faced with new things needed to be learned. Even the most
+seasoned C++ developer will need to ask basic questions about R for example.
+
+When communicating, it's important you mark your communication with appropriate
+tags ([C++], [R], [Ruby] etc.) so it gets noticed by the right people.
+
+Where to get help 👋
+====================
+
+For any question you may have or problems you are facing you can write to 
+user or development :ref:`mailing_list` or you can create an issue on
+:ref:`github`.
+
+For searching through the issues, reporting bugs and creating feature
+requests or proposals you can use :ref:`jira` issue tracker.
+
+.. _jira:
+
+JIRA
+~~~~
+
+When to use JIRA?
+
+- If you want to **report a bug**.
+- If you want to **propose a new feature**.
+- If you want to **propose a bigger change in the documentation**.
+
+In these cases you can create a JIRA issue and connect with other
+contributors in the issue comments. It is good to mention the
+way you are planning to solve the issue and have one of the Arrow
+developers agree with your basic proposal for fixing it. Better
+to ask before you spend too much of your time on something that we
+might think is not a good idea.
+
+.. seealso::
+   How to create a JIRA issue. See `Tips for using JIRA <https://arrow.apache.org/docs/developers/contributing.html#tips-for-using-jira/>`_.
+
+- If you want to **solve an issue that is already in JIRA**, you should
+  connect with other contributors in the issue comments.
+
+JIRA is used to discuss the problem before a Pull Request is possibly made
+(how to start, where to search for the solution, etc..).
+
+.. _github:
+
+GitHub
+~~~~~~
+
+Further options of communicating are provided through GitHub where the project
+is hosted. What we use are GitGub Issues and Pull Requests.
+
+When to use GitHub?
+
+- If you have **a problem with building** one of the Arrow libraries, you can
+  create an issue on GitHub and discuss the possible solution (or write to the
+  user mailing list).
+
+  Making an issue about things you are not sure about may feel intimidating,
+  but it is also useful for others and the project.
+
+  .. note::
+     Make sure to add which operating system and Arrow version you are using in the
+     ticket description plus the debug information/error.
+
+- If you have **a new feature already written**, you can create a Pull Request after
```

Comment:
```suggestion
- If you have **a new contribution already written**, you can create a Pull Request after
```

? (to be more general, since this can also be a bug fix or documentation update)